syftlib/src/main/java/org/openmined/syft/domain/SyftConfiguration.kt

```diff
@@ -55,7 +57,8 @@ class SyftConfiguration internal constructor(
             override val calleeThreadScheduler: Scheduler
                 get() = Schedulers.single()
         }
-        private var socketClient = SocketClient(baseUrl, 20000u, networkingSchedulers)
+        private var retryPolicy = RetryPolicy()
```

Comment:
The retry policy doesn't need to be a user defined policy. Just set exponential backoff to some constant